xsd/netex_part_1/part1_ifopt/netex_ifopt_parking_version.xsd

```diff
@@ -655,6 +655,16 @@ Rail transport, Roads and Road transport
 					<xsd:documentation>Visibible Indication of parking area.  +v1.2.2</xsd:documentation>
 				</xsd:annotation>
 			</xsd:element>
+			<xsd:element name="MonitoredAvailability" type="xsd:boolean" minOccurs="0">
+				<xsd:annotation>
+					<xsd:documentation>Whether real-time global availability data is normally available for the whole PARKING or PARKING AREA.</xsd:documentation>
+				</xsd:annotation>
+			</xsd:element>
```

Comment:
```suggestion
		
```

Reply:
As pointed by Nick, I missed the RealTimeOccupancyAvailable (I was looking in Properties and it was in global information). It's global to the parking (son not specific to areas) but I guess that this is Ok

Reply:
Can the newline be ommited?

Reply:
sure ... I don't know why it was added (I just cut the unwanted part)

Reply:
done